kubernetes/deploy.sh

```diff
@@ -11,20 +11,92 @@ usage: $0 [ -r REVERSE-CIDR ] [ -i DNS-IP ] [ -d CLUSTER-DOMAIN ] [ -t YAML-TEMP
          then the default is to handle all reverse zones (i.e. in-addr.arpa and ip6.arpa)
     -i : Specify the cluster DNS IP address. If not specificed, the IP address of
          the existing "kube-dns" service is used, if present.
+    -u : Applies the translation of kube-dns configmap to the corresponding CoreDNS Corefile configuration.
+
 USAGE
 exit 0
 }
 
 # Simple Defaults
 CLUSTER_DOMAIN=cluster.local
 YAML_TEMPLATE=`pwd`/coredns.yaml.sed
+YAML_UPDATED=`pwd`/coredns.yaml
+
```

Comment:
so we changing the behavior here.
Before : yaml was on the std output
After : yaml is saved into a file.

Reply:
Reading the code, it looks like the $YAML_UPDATED is used as a temporary working file.  

1st step processes $YAML_TEMPLATE to $YAML_UPDATED
2nd step processes $YAML_UPDATED to stdout

But a temporary file like this should be cleaned up when done.  I think a general convention is to make these type of temp files invisible too (start with a dot). However i think its best to avoid if possible creating the temp file at all.